app/Http/Requests/AddSleepRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class AddSleepRequest extends FormRequest
```

Comment:
Why named it as AddSleepRequest? Instead of SleepRequest? This service also being used for update, not only save/add.